### ISSUE TITLE
Create a New Watchlist Tab in Account Tables

### ISSUE THREAD

**Issue by landry314** [bitshares/bitshares-ui#604]
Let's create another tab in the Account tables called "Watchlist" for managing the following of certain assets. This would solve the long standing issue with certain default assets being forced to show up in Account as recently discussed in issue #579.

Almost all stock trading platforms include this feature. Now that we have a nice Portfolio here that tracks current market prices we could easily expand it to include a Watchlist feature in it's own tab as @wmbutler has suggested in issue #247.

Here is an example of a watchlist in action:
https://www.tradingview.com/chart/?symbol=POLONIEX%3ABTSUSD

**Comment by landry314** [comment]
I suggest for default assets in the watchlist:

- bitUSD
- bitCNY
- OPEN.BTC

Anyone want to see others in there?

**Comment by wmbutler** [comment]
Maybe the watchlist could contain every trading pair that is starred on the exchange view for consistency?

**Comment by diogo-gomes** [comment]
> Maybe the watchlist could contain every trading pair that is starred on the exchange view for consistency? - @wmbutler 

**I agree with starred pairs.**

For those who want to trade in EUR and see EUR's pairs growing in volume, BitSharesUI have a _poor_  UX (already miss the EUR tab). This is because the CNY and USD defaults cannot be changed by user and EUR needs to be mixed on `OTHER` tab.

**In my opinion, users must have control on what pairs want to see in watchlist/tabs/favorites/starred.**

**Comment by landry314** [comment]
True, but the volume is too low to spread out right now across too many pairs. I use bitCNY when I would rather use bitUSD just because the volume is concentrated there. Having those default tabs keeps the volume concentrated... it is something to think about... but I do agree that any pair should be able to be in the watchlist.

You can select any pair by typing one of them in the Find Markets in Quote Currency then the other in the Asset Name, then add that pair and star it.

**Comment by wmbutler** [comment]
If you think forcing users to display a pair will increase volume, I'll reply that it's a bad idea for us to acts as agents for particular markets.

**Comment by landry314** [comment]
You are right, Bill, but, just the fact that there are some default pairs in the dashboard or in the watchlist IS promoting some pairs. I think this is good for convenience to the user but possibly a dilemma too.

@diogo-gomes was just complaining that bitEUR markets were hard to get to and he would like to place them in his watchlist, which he will be able to do soon.

**Comment by diogo-gomes** [comment]
> If you think forcing users to display a pair will increase volume, I'll reply that it's a bad idea for us to acts as agents for particular markets. @wmbutler 

I apologize if I mispronounced my point of view. I do not agree with `forcing users to display a pair`. I just find it a bit annoying always see _bitusd_ and _bitcny_ and not be able to see _biteur_. 

Once again, **I do not agree with forcing pairs** like bitEur, I **just think that users should have the possibility to choose the pairs that are visible or not**.

---

> I think bitEUR would be good for a default, the euro is one is the big currencies. also bitGBP, bitJPY, bitRUBLE, maybe even bitCHF or bitINR if they existed. AUD...

I do not think that is a matter of what currency is bigger in trading terms (_but euro is 2º and cny is 8º :smiley:, thanks @landry314 for the link_) but **is kinda unfair see some default currencies that I am forcing to see (bitusd / bitcny)** and not be able to change it. It's not user friendly (it seems one platform only for american and chinese markets and not a global one).

---
**BitShares need to grow and all strategic marketing changes to reach a global public are important.
In my opinion, not have a predefined and unchangeable country currencies is one of them.**

**Comment by Xeldal** [comment]
I think the watchlist tab could be a good addition. It could show a good selection of **basic** market data for a user defined list showing more data than you find in the "Markets" window in "Exchange" tab but without the chart/buy sell/history etc that you find there.    

I don't think this watchlist tab should be where you find "borrow" "settle" etc.  This should be kept in the "My Porfolio" list (hate that tab name btw, should be "My Wallet")  

In addition though, I would add the ability on the "My Porfolio" tab to add assets to the list even if you have a 0 balance.
